src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rfc2898DeriveBytes.cs

```diff
@@ -249,30 +250,45 @@ private void Initialize()
         }
 
         // This function is defined as follows:
-        // Func (S, i) = HMAC(S || i) | HMAC2(S || i) | ... | HMAC(iterations) (S || i) 
+        // Func (S, i) = HMAC(S || i) ^ HMAC2(S || i) ^ ... ^ HMAC(iterations) (S || i) 
         // where i is the block number.
         private byte[] Func()
         {
             byte[] temp = new byte[_salt.Length + sizeof(uint)];
```

Comment:
How big is _salt, or is it arbitrarily large?  Wondering if temp should be a stackalloc.  If not, array pool?